src/app/app.config.ts

```diff
@@ -4,12 +4,14 @@ import { provideHttpClient, withInterceptors } from '@angular/common/http';
 import { routes } from './app.routes';
```

Comment:
Make structure of files same way as we discussed on out meeting. Also please add test for you code. You should cover all public functions in best practises